webgpu.h

```diff
@@ -1125,24 +1125,15 @@ typedef enum WGPUWaitStatus {
     WGPUWaitStatus_Success = 0x00000001,
     /**
      * `0x00000002`.
-     * No WGPUFutures completed within the timeout.
+     * The wait operation succeeded, but no WGPUFutures completed within the timeout.
      */
     WGPUWaitStatus_TimedOut = 0x00000002,
     /**
      * `0x00000003`.
-     * A @ref Timed-Wait was performed when WGPUInstanceFeatures::timedWaitAnyEnable is false.
+     * The call was invalid for some reason (see @ref Wait-Any).
+     * Should produce @ref ImplementationDefinedLogging containing details.
      */
-    WGPUWaitStatus_UnsupportedTimeout = 0x00000003,
-    /**
-     * `0x00000004`.
-     * The number of futures waited on in a @ref Timed-Wait is greater than the supported WGPUInstanceFeatures::timedWaitAnyMaxCount.
-     */
-    WGPUWaitStatus_UnsupportedCount = 0x00000004,
-    /**
-     * `0x00000005`.
-     * An invalid wait was performed with @ref Mixed-Sources.
-     */
-    WGPUWaitStatus_UnsupportedMixedSources = 0x00000005,
+    WGPUWaitStatus_Error = 0x00000003,
```

Comment:
I wonder if we should return this if you try to wait on the null future?

Or on a future that we know is invalid (because it's too big)? (Probably not this one, I don't think we enshrined the monotonically-increasing future id allocation strategy.)

Reply:
Filed under #502